guide/additional-info/changes-in-v14.md

```diff
@@ -4,29 +4,29 @@
 
 v14 requires Node 16.9 or higher to use, so make sure you're up to date. To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it! There are many resources online to help you with this step based on your host system.
 
-### Builders are now included in v14
+### Builders and Rest are now included in v14
 
-If you previously had `@discordjs/builders` manually installed it's _highly_ recommended that you uninstall the package to avoid package naming conflicts.
+If you previously had `@discordjs/builders` and `@discordjs/rest` manually installed it's _highly_ recommended that you uninstall the package to avoid package naming conflicts.
```

Comment:
```suggestion
If you previously had `@discordjs/builders` and `@discordjs/rest` manually installed, it's _highly_ recommended that you uninstall the packages to avoid package naming conflicts.
```